### PR TITLE
Delete instances of pre-specified integrations even if their test failed

### PR DESCRIPTION
**Pull request by bakatzir**
@anara123  Please review the suggested solution as well.

## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/14584

## Description
see related issue.

### CODE REVIEW

File: Tests/conf.json [hunk 1]
```diff
@@ -864,5 +864,11 @@
     },
     "nigthly_integrations": [
         "Lastline"
+    ],
+    "delete_on_failure_integrations": [
+        "Lastline",
+        "VirusTotal",
+        "Joe Security"
     ]
+
```
Comment (ronykoz) [inline Tests/conf.json]: remove

**Comment by ronykoz** [comment]
Looks good!

**Comment by bakatzir** [comment]
add RN to quota. it does delete the instance afterward a test have failed,
thus - phishing inline/attachment do not fail.
screen shot 2018-12-17 at 12 03 09

**Comment by ronykoz** [comment]
Try to add RN section to one of the Phishing playbooks, lets fully test it out

**Comment by bakatzir** [comment]
phishing fails on vt, which is configured. lastline is not running as expected.
reverting the RN in the commits
screen shot 2018-12-17 at 13 36 38

**Pull request closed without merging**